wolfram-library-link/src/args.rs

```diff
@@ -254,6 +256,16 @@ impl<'a> FromArg<'a> for &'a str {
     }
 }
 
+impl FromArg<'_> for char {
+    unsafe fn from_arg(arg: &MArgument) -> Self {
+        String::from_arg(arg).chars().next().unwrap()
+    }
```

Comment:
Hi @sw1sh, as discussed in our meeting, the basic implementation here is exactly right, but we may want to improve the error reporting to help users out in cases where they accidentally pass in strings that are not a single `char` in length.

My proposal would be something like:
```rust

unsafe fn from_arg(arg: &MArgument) -> Self {
    let string = String::from_arg(arg);

    let mut chars = string.chars();

    let Some(char) = chars.next() else {
        panic!("char argument string was empty")
    };

    if chars.next().is_some() else {
        panic!("char argument string contains multiple chars: {string}")
    }
}
```